python/feathub/processors/flink/table_builder/flink_table_builder.py

```diff
@@ -103,6 +103,7 @@ def build(
         keys: Union[pd.DataFrame, TableDescriptor, None] = None,
         start_datetime: Optional[datetime] = None,
         end_datetime: Optional[datetime] = None,
+        bounded_source: bool = False,
```

Comment:
Should we be able to get this information from `features`, which in turn can get this information from e.g. `KafkaSource`?